controllers/serviceController_int_test.go

```diff
@@ -50,34 +57,65 @@ func init() {
 	if err != nil {
 		panic(err)
 	}
+
+	serviceDeployment = &v1.Deployment{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "nexus",
+			Namespace: "my-test-namespace",
+			Labels:    map[string]string{"dogu.name": "nexus"},
+		},
+		Spec: v1.DeploymentSpec{
+			Selector: &metav1.LabelSelector{
+				MatchLabels: map[string]string{"dogu.name": "nexus"},
+			},
+			Template: corev1.PodTemplateSpec{
+				ObjectMeta: metav1.ObjectMeta{
+					Labels: map[string]string{"dogu.name": "nexus"},
+				},
+				Spec: corev1.PodSpec{
+					Containers: []corev1.Container{
+						{Name: "nexus-123123", Image: "nginx:1.7.9"},
+					},
+				},
+			},
+		},
+		Status: v1.DeploymentStatus{
+			Replicas:      0,
+			ReadyReplicas: 0,
+		},
+	}
 }
 
 var _ = Describe("Creating ingress objects with the ingress generator", func() {
-	ctx := context.TODO()
+	ctx := context.Background()
+	resetData()
 
 	Context("Handle new service resource", func() {
 		AfterEach(cleanup)
-
-		It("Should do nothing if service without annotations", func() {
-			By("Creating service with no annotations")
-			Expect(k8sClient.Create(ctx, serviceNoAnnotations)).Should(Succeed())
-
-			By("Expect no ingress resource")
-			expectedIngress := &networking.IngressList{}
-
-			Eventually(func() bool {
-				err := k8sClient.List(ctx, expectedIngress)
-				return err == nil
-			}, timeoutInterval, pollingInterval).Should(BeTrue())
-
-			Expect(len(expectedIngress.Items)).Should(Equal(0))
-		})
+		//
```

Comment:
What happens with this dead code?

Reply:
I commented them out for faster feedback. I commented them back in.

Reply:
They also work :)